modules/simple_forms_api/app/form_mappings/vba_21_4140.json.erb

```diff
@@ -52,43 +52,68 @@
   <%# 10. Were you employed by VA, others or self employed at any time during the past 12 months? %>
   "F[0].Page_1[0].RadioButtonList[0]": "<%= form.data.dig('RadioButtonList') %>",
 
-  "F[0].Page_1[0].Type_Of_Work[0]": "<%= form.data.dig('Type_Of_Work') %>",
-  "F[0].Page_1[0].Time_Lost_From_Illness[0]": "<%= form.data.dig('Time_Lost_From_Illness') %>",
-  "F[0].Page_1[0].Name_And_Address_Of_Employer[0]": "<%= form.data.dig('Name_And_Address_Of_Employer') %>",
-  "F[0].Page_1[0].Hours_Per_Week[0]": "<%= form.data.dig('Hours_Per_Week') %>",
-  "F[0].Page_1[0].Date_Of_Employment_From[0]": "<%= form.data.dig('Date_Of_Employment_From') %>",
-  "F[0].Page_1[0].Date_Of_Employment_To[0]": "<%= form.data.dig('Date_Of_Employment_To') %>",
-  "F[0].Page_1[0].Gross_Earnings_Per_Month[0]": "<%= form.data.dig('Gross_Earnings_Per_Month') %>",
+  <%# Section II - Employment Certification %>
+  <%# 11A. Name and Address of Employer %>
+  "F[0].Page_1[0].Name_And_Address_Of_Employer[0]": "<%= form.employment_history[0].name_and_address %>",
+
+  <%# 11B. Type of Work %>
+  "F[0].Page_1[0].Type_Of_Work[0]": "<%= form.employment_history[0].type_of_work %>",
+
+  <%# 11C. Hours Per Week %>
+  "F[0].Page_1[0].Hours_Per_Week[0]": "<%= form.employment_history[0].hours_per_week %>",
+
+  <%# 11D. Dates of Employment or Self-Employment %>
+  "F[0].Page_1[0].Date_Of_Employment_From[0]": "<%= form.employment_history[0].date_started %>",
+  "F[0].Page_1[0].Date_Of_Employment_To[0]": "<%= form.employment_history[0].date_ended %>",
+
+  <%# 11E. Time Lost From Illness %>
+  "F[0].Page_1[0].Time_Lost_From_Illness[0]": "<%= form.employment_history[0].lost_time %>",
+
+  <%# 11F. Highest Gross Earnings Per Month %>
+  "F[0].Page_1[0].Gross_Earnings_Per_Month[0]": "<%= form.employment_history[0].highest_income %>",
 
   <%# Page 2 %>
   <%# Veteran's Social Security No. %>
   "F[0].#subform[1].Veterans_Social_SecurityNumber_FirstThreeNumbers[0]": "<%= form.ssn[0] %>",
   "F[0].#subform[1].Veterans_Social_SecurityNumber_SecondTwoNumbers[0]": "<%= form.ssn[1] %>",
   "F[0].#subform[1].Veterans_Social_SecurityNumber_LastFourNumbers[0]": "<%= form.ssn[2] %>",
 
+  <%# Section II - Employment Certification (Continued) %>
+  <%# 11A. Name and Address of Employer %>
+  "F[0].#subform[1].Name_And_Address_Of_Employer[0]": "<%= form.employment_history[3].name_and_address %>",
+  "F[0].#subform[1].Name_And_Address_Of_Employer[1]": "<%= form.employment_history[2].name_and_address %>",
+  "F[0].#subform[1].Name_And_Address_Of_Employer[2]": "<%= form.employment_history[1].name_and_address %>",
```

Comment:
Form mappings go from bottom of the page to the top for some reason, so all employers are iterated over in reverse order.

Reply:
Interesting. Thanks for the note.